app/models/pipeline_run.rb

```diff
@@ -1390,4 +1391,35 @@ def outputs_by_step(can_see_stage1_results = false)
   def self.viewable(user)
     where(sample_id: Sample.viewable(user).pluck(:id))
   end
+
+  # Keys here are used as cache keys for report_info action in SamplesController.
+  # The values here are used as defaults for PipelineSampleReport.jsx.
+  def report_info_params
+    {
+      pipeline_version: pipeline_version || PipelineRun::PIPELINE_VERSION_WHEN_NULL,
+      # TODO: (gdingle): is this good enough for cache hits? see get_background_id
+      background_id: sample.default_background_id,
+      # scoring_model is currrently static
+      scoring_model: TaxonScoringModel::DEFAULT_MODEL_NAME,
+      # TODO: (gdingle): why does PipelineSampleReport and SamplesController have different default sort_by?
```

Comment:
@cdebourcy any thoughts about this? the backend default is `highest_nt_aggregatescore`

Reply:
this is only for ranking purposes where the front end can manipulate., so I don't think it really matters.